### PR TITLE
200/201 map info view

### LINKED ISSUE #218: As an attendee, I want to be able to accept an event from the map.

### PR DESCRIPTION
**Pull request by JonatanBonjour**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java [hunk 1]
```diff
@@ -167,4 +165,41 @@ public void handleOnBackPressed() {
...
+                mNumberSwipe = 0;
+            }
+        });
+    }
+
+    private int bundleEventHash() {
+        Bundle bundle = this.getArguments();
+        int eventHash = DEFAULT_VALUE;
+        if (bundle != null) {
+            eventHash = bundle.getInt("eventHash", DEFAULT_VALUE);
```
Comment (ksabaa15) [inline app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java:207]: as long as it works

**Review by ksabaa15 — CHANGES_REQUESTED** [review]
Nice work on the event hash idea, but the back button still doesn't work and it seems that Glide still glitches often. Sometimes it's the default image that pops out and sometimes it's the one of another event. The glide issue should be discussed with the assistants, but the back button should be easily fixed.

File: app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java (hunk 1 above)
Comment (ros-dorian) [inline app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java:207]: A mettre dans UIConstants

File: app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java [hunk 2]
```diff
@@ -167,4 +165,41 @@ public void handleOnBackPressed() {
...
+        }
+        return eventHash;
+    }
+
+    private void goBackToMapIfConditionsAreMet() {
+        if (mArrayAdapter.isEmpty() && bundleEventHash() != DEFAULT_VALUE) {
```
Comment (ros-dorian) [inline app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java]: Pourquoi pas utiliser mEventHash à la place de bundleEventHash ?

File: app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java [hunk 3]
```diff
@@ -167,4 +165,41 @@ public void handleOnBackPressed() {
...
+            getActivity().getSupportFragmentManager().beginTransaction().replace(R.id.main_container, new MapFragment()).commit();
```
Comment (ros-dorian) [inline app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java]: Pas moyen de faire passer cette ligne dans les tests ?

**Comment by JonatanBonjour** [comment]
> Nice work on the event hash idea, but the back button still doesn't work and it seems that Glide still glitches often. Sometimes it's the default image that pops out and sometimes it's the one of another event. The glide issue should be discussed with the assistants, but the back button should be easily fixed.

For the back button, Corentin still need to work on it.

File: app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java (hunk 1 above)
Comment (JonatanBonjour) [reply app/src/main/java/ch/epfl/sdp/ui/main/swipe/SwipeFragment.java:207]: je trouve que c'est pas vraiment lié à l'UI, non ?

**Comment by JonatanBonjour** [comment]
> > Nice work on the event hash idea, but the back button still doesn't work and it seems that Glide still glitches often. Sometimes it's the default image that pops out and sometimes it's the one of another event. The glide issue should be discussed with the assistants, but the back button should be easily fixed.
> 
> For the back button, Corentin still need to work on it.

I have managed to fix it, it's working now.

**Review by MartinBeaussart — APPROVED** [review]

**Review by ksabaa15 — APPROVED** [review]

**Pull request merged**